modules/ocf/manifests/rootpw.pp

```diff
@@ -4,10 +4,10 @@
 # The root password is a last resort that is only used in rare cases where
 # there is no other option.
 #
-# To regenerate the root password, see /opt/share/utils/staff/puppet/gen-rootpw
+# To regenerate the root password, see the gen-rootpw script from utils
```

Comment:
This script needs to be updated (somehow), right?

Reply:
Yeah, this script wouldn't really work any more. Previously it wrote to [a specific file](https://github.com/ocf/utils/blob/13ff417c39eff56a3eebac8d8e8193b7d2113d9d/staff/puppet/gen-rootpw#L5), but this review changes from using that file to putting the data in hiera. The script could be updated to sed something in hierdata, but I think it's probably just better to print out to the terminal and ask the end user to update it themselves. That script is almost never run anyway (hasn't been run to completion in the last 4 years anyway) and updating hiera is generally pretty easy (update a value in a yaml file in the private share) and anyone updating the root password should already know how to do that.

Reply:
I opened https://github.com/ocf/utils/pull/126 for this, so I'm going to resolve this point.